.github/workflows/smoke-test-release.yml

```diff
@@ -690,3 +556,103 @@ jobs:
                     -f env_description="${{ env.ENV_DESCRIPTION }}" \
                     -f test_type="e2e" \
                     --repo woocommerce/woocommerce-test-reports
+
+    test-plugins:
+        name: With ${{ matrix.plugin }}
+        runs-on: ubuntu-20.04
+        needs: [get-tag]
+        env:
+            ALLURE_RESULTS_DIR: ${{ github.workspace }}/plugins/woocommerce/tests/e2e-pw/allure-results
+            ALLURE_REPORT_DIR: ${{ github.workspace }}/plugins/woocommerce/tests/e2e-pw/allure-report
+            ARTIFACT_NAME: E2E test on wp-env with ${{ matrix.plugin }} installed (run ${{ github.run_number }})
+        strategy:
+            fail-fast: false
```

Comment:
Setting `fail-fast`  to false will prevent tests within the same matrix to be cancelled when one of them failed.